super-size/many-extensions/pom.xml

```diff
@@ -10,8 +10,7 @@
     <artifactId>many-extensions</artifactId>
     <name>Quarkus QE TS: Many extensions</name>
     <properties>
-        <!-- set to 7g after we experienced https://github.com/quarkusio/quarkus/issues/44216 -->
-        <quarkus.native.native-image-xmx>10g</quarkus.native.native-image-xmx>
+        <quarkus.native.native-image-xmx>5g</quarkus.native.native-image-xmx>
```

Comment:
This is default value already https://github.com/quarkus-qe/quarkus-test-suite/blob/7a5a184040487cc031a1166e7c8cf600fbe5b236/pom.xml#L703 so setting it here makes no difference?

Reply:
Yes it makes difference, so we can probably remove it entirely.

Reply:
I removed it.

Reply:
Thanks